frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.node-selector.scenario.ts

```diff
@@ -0,0 +1,52 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { createResource, deleteResource, click } from '@console/shared/src/test-utils/utils';
+import * as editNodeSelectorView from '../views/editNodeSelectorView';
+import * as virtualMachineView from '../views/virtualMachine.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { getVMManifest } from './utils/mocks';
+import { getRandStr } from './utils/utils';
+
+describe('KubeVirt VM detail - edit Node Selector', () => {
+  const testVM = getVMManifest('Container', testName, `node-selector-vm-${getRandStr(5)}`);
+  const vm = new VirtualMachine(testVM.metadata);
+
+  beforeAll(async () => {
+    createResource(testVM);
+  });
+
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it(
+    'Adds a Node Selector, then removes it',
```

Comment:
Could you please add ID(CNV-4133) identifier? Thank you

Reply:
not sure what you mean

Reply:
Just naming the test cases like:
'ID(CNV-4133) Adds a Node Selector, then removes it'